specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-11-01-preview/examples/CreateRangerCluster.json

```diff
@@ -0,0 +1,239 @@
+                "13be6971-79db-4f33-9d41-b25589ca25ac"
+              ],
+              "mode": "automatic",
+              "users": [
+                "testuser1@contoso.com",
+                "testuser2@contoso.com"
+              ]
+            }
+          }
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid/resourceGroups/hiloResourcegroup/providers/Microsoft.HDInsight/clusterPools/clusterpool1/clusters/cluster1",
+        "name": "cluster1",
+        "type": "Microsoft.HDInsight/clusterPools/clusters",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "clusterType": "ranger",
+          "deploymentId": "45cd32aead6e4a91b079a0cdbfac8c36",
+          "computeProfile": {
+            "nodes": [
+              {
+                "type": "head",
+                "vmSize": "Standard_D3_v2",
+                "count": 2
+              }
+            ]
+          },
+          "clusterProfile": {
+            "clusterVersion": "0.0.1",
+            "ossVersion": "2.2.3",
+            "components": [
+              {
+                "name": "HDFS",
+                "version": "2.2.3"
+              }
+            ],
+            "identityProfile": {
+              "msiResourceId": "/subscriptions/subid/resourceGroups/hiloResourcegroup/providers/Microsoft.ManagedIdentity/userAssignedIdentities/test-msi",
+              "msiClientId": "de91f1d8-767f-460a-ac11-3cf103f74b34",
+              "msiObjectId": "40491351-c240-4042-91e0-f644a1d2b441"
+            },
+            "authorizationProfile": {
+              "userIds": [
+                "testuser1",
+                "testuser2"
+              ]
+            },
+            "connectivityProfile": {
+              "web": {
+                "fqdn": "cluster1.clusterpool1.westus2.projecthilo.net"
+              }
+            },
+            "rangerProfile": {
+              "rangerAdmin": {
+                "admins": [
+                  "testuser1@contoso.com",
+                  "testuser2@contoso.com"
+                ],
+                "database": {
+                  "host": "testsqlserver.database.windows.net",
+                  "name": "testdb",
+                  "passwordSecretRef": "https://testkv.vault.azure.net/secrets/mysecret/5df6584d9c25418c8d900240aa6c3452",
+                  "username": "admin"
+                }
+              },
+              "rangerAudit": {
+                "storageAccount": "https://teststorage.blob.core.windows.net/testblob"
+              },
+              "rangerUsersync": {
+                "enabled": true,
+                "groups": [
+                  "0a53828f-36c9-44c3-be3d-99a7fce977ad",
+                  "13be6971-79db-4f33-9d41-b25589ca25ac"
+                ],
+                "mode": "automatic",
+                "users": [
+                  "testuser1@contoso.com",
+                  "testuser2@contoso.com"
+                ]
```

Comment:
is it privacy reviewed?

Reply:
yes. This property has been onboarded in previous apiversion.